index.php

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+      
+ <title><?php echo "Assignment 7"; ?></title>
+  <meta charset="utf-8"> 
+</head>
+<body>
+
+<?php 
+$meatloaf = 'Rice';
+echo $meatloaf;
+echo "<br>";
+$my_Num = 2; 
+echo $my_Num;
+echo "<br>";
+$txt = 'Red Rocks Park';
+echo $txt;
+echo "<br>";
+$chickenhawk = 'Rabbit';
+echo $chickenhawk;
+?>
+
+
+
+<h1>Good morning, Nebraska!</h1>
+<div>Last night I ate <?php echo $meatloaf ?>.</div>
```

Comment:
these are the outputs I want to see! Nice work! You got those variables doing their thing!